packages/components/src/util/typography.tsx

```diff
@@ -54,15 +56,20 @@ export type TypographyProps<IComponent extends React.ElementType> = {
    * we will add stricter enforcement -- e.g. enforcing a boldness
    * for each size or checking for mutually exclusive props.
    */
-  weight: "bold" | "normal" | null;
+  weight?: "bold" | "normal" | null;
+  /**
+   * Specifies the bottom-margin that should be applied to the typography element
+   */
+  spacing?: TypographyMargin;
```

Comment:
could we actually handle this similarly to `weight`, where there's a default `mb-0` set on all typography, and extra styles / className only gets passed if it's explicitly set?

Reply:
was considering that, sounds good